tools/batch-script/batch.go

```diff
@@ -0,0 +1,187 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package batch_script
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-go-sdk"
+	flowclient "github.com/onflow/flow-go-sdk/client"
+	"github.com/rs/zerolog"
+	"google.golang.org/grpc"
+)
+
+// getFlowClient initializes and returns a flow client
+func getFlowClient(flowClientUrl string) *flowclient.Client {
+	flowClient, err := flowclient.New(flowClientUrl, grpc.WithInsecure())
+	if err != nil {
+		panic(err)
+	}
+	return flowClient
+}
+
+// Config defines that application's config
+type Config struct {
+	// BachSize is the number of addresses for which to run each script
+	BatchSize int
+	// ContinualPoll enables the script to continue polling Flow at the most recent block height after
+	// it is completed the current block height. If AtBlockHeight is set to a value greater than 0,
+	// ContinualPoll is ignored (it will be treated as false).
+	ContinualPoll bool
```

Comment:
This is not used. This is a leftover from the fraud-worker code.

Reply:
Good catch! Removed in 4c4a1239b97aee32b4e0fe333f1b971e43c641d3